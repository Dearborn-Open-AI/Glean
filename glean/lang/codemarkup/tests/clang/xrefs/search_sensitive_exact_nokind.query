query: codemarkup.search.SearchByName { searchcase = Sensitive, name = "foo" }
perf: true
transform: [gensort, []]
