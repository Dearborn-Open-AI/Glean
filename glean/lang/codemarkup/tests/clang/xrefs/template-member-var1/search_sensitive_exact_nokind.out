[
  "@generated",
  {
    "key": {
      "searchcase": 0,
      "name": "foo",
      "entity": {
        "cxx": {
          "defn": {
            "variable": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "foo" },
                    "scope": {
                      "recordWithAccess": {
                        "record": {
                          "key": {
                            "name": { "key": "A" },
                            "scope": { "global_": { } }
                          }
                        },
                        "access": 0
                      }
                    }
                  }
                },
                "type": { "key": "int" },
                "kind": {
                  "global_": { "kind": 2, "attribute": 0, "definition": true }
                },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 32,
                  "columnBegin": 1,
                  "lineEnd": 32,
                  "columnEnd": 34
                }
              }
            }
          }
        }
      },
      "location": {
        "name": "foo",
        "file": { "key": "test.cpp" },
        "location": {
          "range": {
            "file": { "key": "test.cpp" },
            "lineBegin": 32,
            "columnBegin": 1,
            "lineEnd": 32,
            "columnEnd": 34
          }
        },
        "span": { "start": 632, "length": 3 }
      },
      "kind": 14,
      "language": 0
    }
  },
  {
    "key": {
      "searchcase": 0,
      "name": "foo",
      "entity": {
        "cxx": {
          "defn": {
            "variable": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "foo" },
                    "scope": {
                      "recordWithAccess": {
                        "record": {
                          "key": {
                            "name": { "key": "A" },
                            "scope": { "global_": { } }
                          }
                        },
                        "access": 0
                      }
                    }
                  }
                },
                "type": { "key": "int" },
                "kind": {
                  "global_": { "kind": 2, "attribute": 0, "definition": true }
                },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 39,
                  "columnBegin": 1,
                  "lineEnd": 39,
                  "columnEnd": 15
                }
              }
            }
          }
        }
      },
      "location": {
        "name": "foo",
        "file": { "key": "test.cpp" },
        "location": {
          "range": {
            "file": { "key": "test.cpp" },
            "lineBegin": 39,
            "columnBegin": 1,
            "lineEnd": 39,
            "columnEnd": 15
          }
        },
        "span": { "start": 777, "length": 3 }
      },
      "kind": 14,
      "language": 0
    }
  }
]